src/lib.rs

```diff
@@ -0,0 +1,17 @@
+extern crate crossbeam_channel;
+extern crate humansize;
+extern crate num_cpus;
+extern crate rayon;
+
+pub mod walk;
+
+pub use walk::Walk;
+use humansize::{file_size_opts, FileSize};
+
+pub fn print_result(size: u64) {
```

Comment:
I don't think the `print_result` function should be part of the library. This is very much application specific.